etc/pr_check/go.mod

```diff
@@ -0,0 +1,21 @@
+module pachyderm/pr_check
```

Comment:
I'd like to get away from nested go modules if we can; they are technically not allowed and gopls complains.  I think we should make a src/internal/tools/ directory and do one subdirectory per project.  (As we clean things up, most of etc/ will move into src/internal/tools/.  As an aside, we should also have src/internal/experimental/username/  directories where username doesn't require code review to commit there ;)  All of this has to be under src/internal so that outside parties don't depend on the code and then get mad when it's deleted/changed/broken/etc.